confidential/confidential.go

```diff
@@ -492,7 +516,7 @@ func inAssetGenerators(inAssets, inAssetBlinders [][]byte) ([]*secp256k1.Generat
 
 	inGenerators := make([]*secp256k1.Generator, 0, len(inAssets))
 	for i, inAsset := range inAssets {
-		gen, err := secp256k1.GeneratorGenerateBlinded(
+		gen, err := secp256k1.GeneratorGenerateBlinded( //TODO in elements repo generate_blinded is not used?
```

Comment:
it's used indeed https://github.com/ElementsProject/elements/blob/d973e1105d0eacda1b31316c1a58aec4a73029b2/src/blind.cpp#L348.